test/suite0.janet

```diff
@@ -33,7 +33,14 @@
 (buffer/clear b)
 (with-dyns [:out b]
   (fmt/format-print "# a comment"))
-(pp b)
 (assert (deep= b @"# a comment\n\n") "format-print only comment")
 
+(buffer/clear b)
```

Comment:
Well, I am using `assert-error` for this, and rarely do I check the error message. I would go with:

```
(assert-error  "format-print errors with unbalanced parenthesis" 
                     (capture-stdout (fmt/format-print "print )"))
```

Reply:
I don't think this is good practice. The function under test could error for a number of reasons and if you're only checking whether an error occurred and not whether it was the expected error, I'm not sure how much confidence you can really have in that test.

Reply:
That is indeed true, but your code does not seem to do it either. And spork tests already go this error blind path. 

Anyway, your call! 

And thank you very much for the test for #14.

Reply:
Yep, can continue the discussion about error messages in #24.